### DISCUSSION TITLE
Unable to setup maestro in windows

### DISCUSSION THREAD

**Discussion by LalithKumar702** [mobile-dev-inc/Maestro#292] [Q&A] [answered]
downloaded CLI 1.10.1 for windows. 
unzipped and opened bin folder in terminal.
created a file named flow.yaml in bin folder itself
Now, while i am running the script maestro test flow.yaml, a error is occuring continously. what can i do to remove that error?

error description:
The input line is too long. 
The syntax of the command is incorrect

![Screenshot 2022-10-19 192531](https://user-images.githubusercontent.com/57323345/196711574-7fdcbfea-1974-417f-8a8e-cba3eea5d25d.png)

**Comment by axelniklasson** [comment, marked as answer]
Hi there @LalithKumar702! Docs for Windows installation can be found [here](https://maestro.mobile.dev/getting-started/installing-maestro/windows).
